packages/shared/src/components/auth/AuthExtensionBanner.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { ReactElement } from 'react';
+import { BottomBannerContainer } from '../banners';
+import { OnboardingHeadline } from './OnboardingHeadline';
+import { MemberAlready } from '../onboarding/MemberAlready';
+import { AuthTriggers } from '../../lib/auth';
+import { useAuthContext } from '../../contexts/AuthContext';
+import { GetExtensionButton } from '../buttons/GetExtensionButton';
+import { cloudinaryAuthBannerBackground as bg } from '../../lib/image';
+import { Image } from '../image/Image';
+
+const laptopBg = bg.replace('/f_auto', '/c_auto,g_center,w_1440/f_auto/');
+const desktopBg = bg.replace('/f_auto', '/c_auto,g_center,w_1920/f_auto/');
```

Comment:
Why don't are we using an object for this? Maybe is better to not replace it dynamically every time

```ts
const bannerBg = { default: '', laptop: '', desktop: '' };
```

Reply:
I would just introduce different consts in `src/lib/image.ts`

Reply:
replacing can break fast if links change for any reason